modules/microshift-fips-rpm-system.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * microshift_install/microshift-install-rpm.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-fips-rpm-system_{context}"]
+= Enabling FIPS compliance with RPM-based installations
+
+Using FIPS with {product-title-first} requires enabling the cryptographic module self-checks in your {op-system-base-full} installation. After the host operating system has been set up to start up with the FIPS modules, {microshift-short} containers are automatically enabled to run in FIPS mode.
+//Q: the RHEL docs say podman does this; true for MicroShift?
+
+* When {op-system-base} is booted in FIPS mode, {microshift} core components use the RHEL cryptographic libraries that have been submitted to NIST for FIPS 140-2/140-3 Validation on only the x86_64 architectures.
+
+* You must enable FIPS mode when you install {op-system-base} {op-system-version-major} on the machines that you plan to use as worker machines.
+
+[IMPORTANT]
+====
+Because FIPS must be enabled before the operating system that your cluster uses boots for the first time, you cannot enable FIPS after you deploy a cluster.
+====
+
+* {microshift-short} uses a FIPS-compatible golang compiler.
+
+* FIPS is supported in the CRI-O container runtime.
+//OCP has a table of attributes and limitations that didn't make a whole lot of sense; do we need to add any limitation or caveat to cri-o container runtime support?
+
+== Limitations
+
+* TLS implementation FIPS support is not complete.
```

Comment:
See IBM SG entry on "complete". "finished" or "completed" might be better here, or maybe alternatively "is in progress", if accurate?

Reply:
That is straight out of the OCP docs (that haven't been touched in a long time). So I truly don't know.